src/Makefile

```diff
@@ -99,10 +109,13 @@ $(LIBTRIEMAP):
 .PHONY: $(LIBTRIEMAP)
 
 BLAS_CFLAGS = -fPIC -I $(abs_path)/util -include rmalloc.h -DREDIS_MODULE_TARGET -DGB_MALLOC=rm_malloc -DGB_CALLOC=rm_calloc -DGB_REALLOC=rm_realloc -DGB_FREE=rm_free -DGBCOMPACT
-$(GRAPHBLAS):
-	$(MAKE) CMAKE_OPTIONS="-DJOBS=32 -DUSER_POSIX=1 -DCMAKE_CXX_FLAGS=-std=c++11 -DCMAKE_C_FLAGS='$(BLAS_CFLAGS)'" static_only -C ../deps/GraphBLAS
 
-.PHONY: $(GRAPHBLAS)
+# The target libgraphblas.a required by redisgraph.so
+# is conditionaly dependent on the existence of the library libgraphblas.a
+# incase the file doesn't exists we'll build it, otherwise this rule is skipped.
+# see Makefile "order-only dependencies".
+$(GRAPHBLAS): | $(GRAPHBLAS)
```

Comment:
This doesn't work for me:
`make[1]: Circular ../deps/GraphBLAS/build/libgraphblas.a <- ../deps/GraphBLAS/build/libgraphblas.a dependency dropped.`

Reply:
same here, let's discus